gui/default/syncthing/folder/revertOverrideView.html

```diff
@@ -0,0 +1,33 @@
+<modal id="revert-override-confirmation" status="danger" icon="fas fa-question-circle" heading="{{revertOverrideParams.heading}}" large="no" closeable="yes">
+  <div class="modal-body" ng-switch="revertOverrideParams.type">
+    <div ng-switch-when="deleteEnc">
+      <p>
+        <span translate>Unexpected items have been found in this folder.</span></br>
+        <span translate translate-value-receive-encrypted="{{'Receive Encrypted' | translate}}">You should never add or change anything locally in a "{%receiveEncrypted%}" folder.</span>
+      </p>
+      <p translate>
+        Are you sure you want to permanently delete all these files?
+      </p>
+    </div>
+    <div ng-switch-default>
+      <p>
+        <span translate>Warning</span>:</br>
+        <span ng-switch="revertOverrideParams.type">
+          <span ng-switch-when="override" translate>The folder content on other devices will be overwritten to become identical with this device. E.g. files not present here will be deleted on other devices.</span>
```

Comment:
```suggestion
          <span ng-switch-when="override" translate>The folder content on other devices will be overwritten to become identical with this device, e.g. files not present here will be deleted on other devices.</span>
```

The "E.g." at the beginning of the second sentences looks kind of weird. How about using a comma, or if the sentence becomes too long, then just starting the second sentence with writing "for example" or "for instance" in full?

Reply:
I think the second sentence is fine without the e.g. as well, just saying that files here/there will be deleted.